libkbfs/journal_server.go

```diff
@@ -93,26 +105,51 @@ func makeJournalServer(
 	return &jServer
 }
 
+func (j *JournalServer) getDirLocked() string {
+	if j.currentVerifyingKey == (VerifyingKey{}) {
+		panic("currentVerifyingKey is zero")
+	}
+
+	// Device IDs and verifying keys are globally unique, so no
+	// need to have the uid in the path. Furthermore, everything
+	// after the first two bytes (four characters) are randomly
+	// generated, so taking the first 36 characters gives ~1/2^64
+	// collision probability.
```

Comment:
Yeah, I thought about it. I decided against it, though, because it would be annoying to have to convert IDs from hex to base32/62 when debugging.